monai/apps/reconstruction/transforms/dictionary.py

```diff
@@ -229,6 +230,9 @@ def __call__(self, data: Mapping[Hashable, Tensor]) -> dict[Hashable, Tensor]:
         Returns:
             the new data dictionary
         """
+        if lazy is True:
```

Comment:
```suggestion
        if lazy:
```

Reply:
Same thing:

```
foo = 123
if foo:
  print("foo")
if foo is True:
  print("foo")
else:
  print("bar")
```
This is why I try to avoid using if x: unless I want those semantics